File: impl/php/examples/basic_usage.php
<?php

declare(strict_types=1);

require_once __DIR__ . '/../src/MakrellFormatException.php';
require_once __DIR__ . '/../src/MiniMbf.php';
require_once __DIR__ . '/../src/Mron.php';

use Makrell\Formats\Mron;

$doc = Mron::parseString('name Makrell features [comments trailing-commas typed-scalars] stable false');

var_export($doc);
echo PHP_EOL;

File: impl/jvm/examples/BasicUsage.java
package dev.makrell.formats.examples;

import dev.makrell.formats.Mron;

import java.util.Map;

public final class BasicUsage {
    public static void main(String[] args) {
        @SuppressWarnings("unchecked")
        Map<String, Object> doc = (Map<String, Object>) Mron.parseString(
            "name Makrell features [comments \"trailing-commas\" \"typed-scalars\"] stable false"
        );

        System.out.println(doc.get("name"));
        System.out.println(doc.get("features"));
        System.out.println(Mron.writeString(doc));
    }
}
